### PR TITLE
[EWS O365] support non-ascii chars

### PR DESCRIPTION
**Pull request by michal-dagan**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/XSUP-26626)

## Description
- Fixed an issue where parsing attachments with non-ASCII encoded characters failed.
- Added support for the sender name in the incident parsing.
- Increased the MAX_INCIDENTS_PER_FETCH to 200

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/MicrosoftExchangeOnline/Integrations/EWSO365/EWSO365_test.py [hunk 1]
```diff
@@ -533,6 +533,37 @@ def test_parse_incident_from_item():
...
+def test_parse_incident_from_item_non_ascii(mime_content):
+    """
+    Given:
+        - Message item with attachment that contains non-ASCII characters or only ASCII characters.
+
+    When:
+        - Parsing incident from item
+
+    Verify:
+        - Parsing runs successfully
```
Comment (bziser) [inline Packs/MicrosoftExchangeOnline/Integrations/EWSO365/EWSO365_test.py:549]: Can we check in the test that the parsing ran successfully and the output is what we want?

**Review by bziser — APPROVED** [review]
great job!

**Review by bziser — APPROVED** [review]

File: Packs/MicrosoftExchangeOnline/Integrations/EWSO365/EWSO365.yml [hunk 2]
```diff
@@ -79,7 +79,7 @@ configuration:
   type: 0
   section: Collect
   required: false
-- display: Max incidents per fetch (up to 50)
+- display: Max incidents per fetch (up to 200). Performance might be affected by a higher than 50.
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftExchangeOnline/Integrations/EWSO365/EWSO365.yml:82]:
```suggestion
- display: Maximum number of incidents per fetch (up to 200). Performance might be affected by a value higher than 50.
```

**Comment by ShirleyDenkberg** [comment]
@bziser @amshamah419 @thefrieddan1 Doc review completed.

**Comment by michal-dagan** [comment]
This PR needs a force merge due to a test playbook failure on XSOAR NG. There is already an open ticket for this issue: 
https://jira-hq.paloaltonetworks.local/browse/CRTX-91323

**Pull request merged**